pydruid/db/api.py

```diff
@@ -328,7 +328,7 @@ def rows_from_chunks(chunks):
         rows = body[:boundary].lstrip('[,')
         body = body[boundary:]
 
-        for row in json.loads('[{rows}]'.format(rows=rows)):
+        for row in json.loads('[{rows}]'.format(rows=rows), object_pairs_hook=OrderedDict):
```

Comment:
You're getting a "Line too long" linting error that breaks the build.

Reply:
Thanks for point out, I have modified the code style to fix the listing error.